packages/flutter_tools/lib/src/base/utils.dart

```diff
@@ -90,7 +90,7 @@ String getElapsedAsMilliseconds(Duration duration) {
 
 /// Return a String - with units - for the size in MB of the given number of bytes.
 String getSizeAsMB(int bytesLength) {
-  return '${(bytesLength / (1024 * 1024)).toStringAsFixed(1)}MB';
+  return '${(bytesLength / (1000 * 1000)).toStringAsFixed(1)}MB';
```

Comment:
Windows uses mebibytes for disk storage. When building a test app, `flutter build apk` reported a size of `16.9MB`, but Windows reported a size of `16.0MB`:

![image](https://github.com/flutter/flutter/assets/737941/a18beee7-1621-4d51-8075-af89477503d3)

16,871,781/1024/1024 = 16.0
16,871,781/1000/1000 = 16.9

Would it be better instead to stick to mebibytes and change the string's unit to `MiB`?

Reply:
Yeah, I would expect this to be MiB

Reply:
What about having it report the correct size based on the OS (MiB on Windows and Linux and MB on macOS). That way the tool reported size always looks like the OS' reported size

Reply:
> What about having it report the correct size based on the OS (MiB on Windows and Linux and MB on macOS). That way the tool reported size always looks like the OS' reported size

This makes sense, let's just make sure we report the correct unit.